stan/math/rev/arr/functor/coupled_ode_system.hpp

```diff
@@ -323,14 +322,16 @@ struct coupled_ode_system<F, var, double> {
 
   /**
    * Returns the initial state of the coupled system.
-   *
+   * 
    * <p>Because the starting state is unknown, the coupled system
    * incorporates the initial conditions as parameters.  The
    * initial conditions for the coupled part of the system are set
    * to zero along with the rest of the initial state, because the
    * value of the initial state has been moved into the
    * parameters.
    *
+   * FIXME(syclik): this documentation is incorrect.
```

Comment:
@wds15, can you update this doc? It's clearly wrong; it states that all the elements should be 0 but are 1 instead.